packages/garbo/src/tasks/dailyItems.ts

```diff
@@ -80,6 +80,23 @@ function drawBestCards(): void {
   }
 }
 
+function bestLeafBurn(): Item {
+  const ignored = $items`lit leaf lasso, day shortener`; // Ignore limited purchases
+  const leafItems = [...BurningLeaves.burnFor]
+    .filter(
+      (entry): entry is [Item, number] =>
+        entry[0] instanceof Item &&
+        entry[0].tradeable &&
+        !ignored.includes(entry[0]),
+    )
+    .map(([item]) => item);
+  const valuePerLeaf = (leafItem: Item): number => {
+    const leafCost = BurningLeaves.burnFor.get(leafItem)!;
+    return garboValue(leafItem) / leafCost;
+  };
+  return maxBy(leafItems, valuePerLeaf);
```

Comment:
```suggestion
  const leafItems = [...BurningLeaves.burnFor]
    .filter(
      (entry): entry is [Item, number] =>
        entry[0] instanceof Item &&
        entry[0].tradeable &&
        !ignored.includes(entry[0]),
    );
  const valuePerLeaf = ([item, leaves]: [Item, number])=> garboValue(item) / leaves;
  return maxBy(leafItems, valuePerLeaf)[0];
```
We should enable the thing in the linter to yell about non-nullish assertions